spec/features/homepage_spec.rb

```diff
@@ -0,0 +1,7 @@
+require "spec_helper"
+feature 'Show text on homepage' do
+  scenario '12 Question App with bootstrap' do
+    visit '/'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.